backend/scripts/seed-local.sql

```
@@ -0,0 +1,244 @@
+  1000000::bigint AS id_base;
+
+-- 規模の値を psql 変数へ取り込む(以降 :n_users のように埋め込んで使う)。
+SELECT n_users, n_courses, chapters_per_course, submissions_per_user, notes_per_user, activity_days
+  FROM _cfg \gset
+
+\echo '=== seed-local: size =' :'size' '/ users =' :n_users '/ submissions_per_user =' :submissions_per_user
+
+-- 依存の子から消す(FK が無くても順序は揃えておく)。
+BEGIN;
+
+DELETE FROM user_daily_activities  WHERE user_id >= 1000000;
```

SQLFluff (4.2.2)

[error] The 'WHERE' keyword should always start a new line.

(LT14)